test/parallel/test-repl-underscore.js

```diff
@@ -195,9 +195,8 @@ function testError() {
       /setImmediate/,
       /^    at/,
       /^    at/,
-      /^    at/,
-      /^    at/,
     ];
+    console.log(lines);
```

Comment:
Is the `console.log(lines)` part of the test? If so, please leave a comment indicating so, if no, please remove it :-)

Reply:
I am pretty sure I had a PR that improved the logging of this in case of a failure. Currently it's hard to know what failed without this. I think I closed that one by accident when I wanted to land it :rofl: So I'll open another one for that.

Reply:
Nope. I’ll remove.